cmd/terramate/cli/cli_runner_test.go

```diff
@@ -116,6 +116,8 @@ func assertRun(t *testing.T, got runResult) {
 func assertRunResult(t *testing.T, got runResult, want runExpected) {
 	t.Helper()
 
+	assert.EqualInts(t, want.Status, got.Status, "exit status mismatch")
```

Comment:
I think checking the status later is better because you need to see the stdout/stderr to understand the issue...

Reply:
Well when it failed for me it was worse because the output check then made no sense at all without me knowing that the exit status was 0 instead of error....maybe we need to do all checks with Error to avoid stopping on the first error that happens here, felt like I had only half the information and on my case it was the half that helped me less.